### ISSUE TITLE
Max Res of Video Thumbnail

### ISSUE THREAD

**Issue by Xenodirt** [talmobi/yt-search#65]
Is there any way to get the max resolution of the video thumbnail?

**Comment by talmobi** [comment]
@Xenodirt https://github.com/talmobi/yt-search/blob/master/src/index.js#L1428-L1429

It should default to the high quality version -- I'm not sure if there's a higher quality version available -- let me know if there is maybe we can implement it -- that being said -- if you know the thumbnail url maybe you only have to change the last part to something else like instead of ".../hqdefault.jpeg" it would become ".../maxdefault.jpeg" or something. I'm unsure.

**Comment by Xenodirt** [comment]
The URL of the thumbnail is always ".../hqdefault.jpeg" instead of ".../maxdefault.jpeg" for me. I tried it with multiple videos, all I got were ".../hqdefault.jpeg" for both the image and thumbnail objects.

**Comment by talmobi** [comment]
@Xenodirt yes currently the code does that -- I don't know if "../maxdeafult.jpeg" is a real image -- it was just something I thought might exist? I don't know if it's possible to get a higher quality thumbnail than "../hqdefault.jpeg" -- do you know if there is?
